database/migrations/2021_02_20_130610_create_commentaires_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateCommentairesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('commentaires', function (Blueprint $table) {
```

Comment:
name of variables and tables should be on the same language, use english for all
rename the table to comments